app/scripts/modules/core/src/application/service/InferredApplicationWarningService.tsx

```diff
@@ -1,5 +1,15 @@
+import React from 'react';
+
 import { Application } from '../application.model';
 import { NotifierService } from 'core/widgets/notifier/notifier.service';
+import { Markdown } from 'core/presentation';
+
+const ConfigureApplicationMessage = ({ appName }: { appName: string }) => (
+  <Markdown
+    message={`The application <b>${appName}</b> has not been <a href="#/applications/${appName}/config">configured</a>.`}
+    options={{ ADD_ATTR: ['onclick'] }}
```

Comment:
can this `options` prop specifically be removed (i see why the other usages need it)?